komal/Learnt-useState/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react'
+
+const App = () => {
+  const state = useState()
+
+  const [Count, setCount] = useState(0)
+
+  const IncNum = () => {
```

Comment:
```suggestion
  const incNum = () => {
```
Function names should also begin with a lowercase letter.

Reply:
okay